vertx-web/src/main/java/io/vertx/ext/web/handler/sockjs/impl/WebSocketTransport.java

```diff
@@ -94,20 +93,8 @@
     WebSocketListener(ServerWebSocket ws, SockJSSession session) {
       this.ws = ws;
       this.session = session;
-      ws.handler(data -> {
-        if (!session.isClosed()) {
-          String msgs = data.toString();
-          if (msgs.equals("")) {
-            //Ignore empty frames
-          } else if ((msgs.startsWith("[\"") && msgs.endsWith("\"]")) ||
-                     (msgs.startsWith("\"") && msgs.endsWith("\""))) {
-            session.handleMessages(msgs);
-          } else {
-            //Invalid JSON - we close the connection
-            close();
-          }
-        }
-      });
+      ws.binaryMessageHandler(data -> handleMessages(data.toString()));
```

Comment:
I think this is not correct, sockjs is text only, right? (this handler should not be set imo)